queue/init.lua

```diff
@@ -21,9 +21,21 @@ local function register_driver(driver_name, tube_ctr)
     queue.driver[driver_name] = tube_ctr
 end
 
+local pre_cfg = {}
```

Comment:
* Add a commentary on why this is needed.
* I don't really like the names `pre_cfg` and `pre_set`, I think it's better to discuss the variants by voice,

Reply:
renamed to deferred_opts/deferred_cfg